src/pl/senla/task9/ex2/Main.java

```diff
@@ -0,0 +1,13 @@
+package pl.senla.task9.ex2;
+
+import java.util.concurrent.Semaphore;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+        new Thread(new MyClass(new Semaphore(1))).start();
```

Comment:
Read again about semaphore, you probably get it wrong.
You passing to your thread different semaphores, so this should not work as task implies. 
Try to reduce sleep() to 1mc in thread and the issue will probably appears in logs, coz threads not really synchronized

try this after:
```
Semaphore semaphore = new Semaphore(1, true);

new Thread(new MyClass(semaphore)).start();
 new Thread(new MyClass(semaphore)).start();
```

Reply:
clear :)
changed to using of 1 semapfore.
reduced sleeping time to 1 ms.
Everything works correctly.